cltk/tokenize/word.py

```diff
@@ -4,30 +4,31 @@
               'Kyle P. Johnson <kyle@kyle-p-johnson.com>',
               'Natasha Voake <natashavoake@gmail.com>',
               'Clément Besnier <clemsciences@aol.com>',
-              'Andrew Deloucas <adeloucas@g.harvard.edu>']
+              'Andrew Deloucas <adeloucas@g.harvard.edu>',
+              'Todd Cook <todd.g.cook@gmail.com>']
 # Author info for Arabic?
 
 __license__ = 'MIT License. See LICENSE.'
 
-from typing import List, Dict, Tuple, Set, Any, Generator
-from abc import abstractmethod
-
+import logging
 import re
+from abc import abstractmethod
+from typing import List, Dict, Tuple, Set, Any, Generator
 
 from nltk.tokenize.punkt import PunktSentenceTokenizer, PunktParameters
-
 from nltk.tokenize.treebank import TreebankWordTokenizer
 
 import cltk.corpus.arabic.utils.pyarabic.araby as araby
-from cltk.tokenize.latin.sentence import LatinPunktSentenceTokenizer
 from cltk.tokenize.greek.sentence import GreekRegexSentenceTokenizer
-
 from cltk.tokenize.akkadian.word import tokenize_akkadian_words, tokenize_akkadian_signs
-
 from cltk.tokenize.middle_english.params import MiddleEnglishTokenizerPatterns
 from cltk.tokenize.middle_high_german.params import MiddleHighGermanTokenizerPatterns
 from cltk.tokenize.old_norse.params import OldNorseTokenizerPatterns
 from cltk.tokenize.old_french.params import OldFrenchTokenizerPatterns
+from cltk.tokenize.latin.word import WordTokenizer as LatinWordTokenizer
+
+LOG = logging.getLogger(__name__)
```

Comment:
We have the CLTK logger which writes to `~/cltk_data/cltk.log`: https://github.com/cltk/cltk/blob/master/cltk/utils/cltk_logger.py

Use example: https://github.com/cltk/cltk/blob/8de70a881befa2686bc106d1108783ab3bda004a/cltk/utils/file_operations.py#L7

https://github.com/cltk/cltk/blob/8de70a881befa2686bc106d1108783ab3bda004a/cltk/utils/file_operations.py#L40

Reply:
I suppose it depends on what we want CLTK to be. 
If CLTK is a library, it should probably log to the null handler since it will inherit app config on a logger.  
If CLTK is an application, then logging to file is okay.
I think we should stay away from log to file method since there is not way to turn off logging for a particular class; we should use Python's built in logging mechanism.

Reply:
> If CLTK is a library
> If CLTK is an application

I haven't thought of logging in a library v application context. Or at least not in a long time. Do you have any references on this?

> I think we should stay away from log to file method since there is not way to turn off logging for a particular class; we should use Python's built in logging mechanism.

What else would a user do with an error other than write it to file? And if they're writing it to file, would it really be too push of us to insist that it be at `cltk_data/cltk.log`? Just wondering, I guess, the example of the kind of self-defined logging you have in mind.

Reply:
reading https://docs.python-guide.org/writing/logging/#logging-in-a-library

```
Logging in a Library¶
Notes for configuring logging for a library are in the logging tutorial. Because the user, not the library, should dictate what happens when a logging event occurs, one admonition bears repeating:

Note
It is strongly advised that you do not add any handlers other than NullHandler to your library’s loggers.
```

OK, case closed! I will open a ticket about this, since the logging all around should be consistent. Thx